server/lib/payments.js

```diff
@@ -205,7 +204,11 @@ export async function createRefundTransaction(transaction, refundedPaymentProces
 
   if (transaction.data?.isFeesOnTop) {
     const feeOnTopTransaction = await models.Transaction.findOne({
-      where: { ...FEES_ON_TOP_TRANSACTION_PROPERTIES, type: 'CREDIT' },
```

Comment:
Are there any erroneous transactions introduced by this query that we would need to fix in the DB?

Reply:
Nope, just the one we manually fixed last week (currently soft-deleted in our DB).